File: cs/AHResultDto.cs
using System;
using System.Collections.Generic;
using ProtoBuf;
[ProtoContract]
public class AHResultDto : IProtostuff {
	[ProtoMember(1)]
	public long Num{get;set;}
	[ProtoMember(2)]
	public int Result{get;set;}
	[ProtoMember(3)]
	public int OddEnven{get;set;}
	[ProtoMember(4)]
	public long LastTimeGrantAward{get;set;}
	[ProtoMember(5)]
	public List<int> CardIds{get;set;}
}

File: src/main/java/org/zgl/rooms/always_happy/dto/AHResultDto.java
package org.zgl.rooms.always_happy.dto;

import org.zgl.utils.builder_clazz.ann.Protostuff;

import java.util.List;

/**
 * @作者： big
 * @创建时间： 2018/5/24
 * @文件描述：
 */
@Protostuff
public class AHResultDto {
    /**第几场*/
    private long num;
    /**出牌结果*/
    private int result;
    /**本局单双 8双 9单*/
    private int oddEnven;
    /**上期奖励*/
    private long lastTimeGrantAward;
    /**牌的id*/
    private List<Integer> cardIds;

    public long getNum() {
        return num;
    }

    public void setNum(long num) {
        this.num = num;
    }

    public int getResult() {
        return result;
    }

    public void setResult(int result) {
        this.result = result;
    }

    public int getOddEnven() {
        return oddEnven;
    }

    public void setOddEnven(int oddEnven) {
        this.oddEnven = oddEnven;
    }

    public long getLastTimeGrantAward() {
        return lastTimeGrantAward;
    }

    public void setLastTimeGrantAward(long lastTimeGrantAward) {
        this.lastTimeGrantAward = lastTimeGrantAward;
    }

    public List<Integer> getCardIds() {
        return cardIds;
    }

    public void setCardIds(List<Integer> cardIds) {
        this.cardIds = cardIds;
    }
}
